### ISSUE TITLE
Multiple parameters with properties not working

### ISSUE THREAD

**Issue by manuelcabral** [jsdoc3/jsdoc#901]
Using multiple parameters with properties does not seem to work. Example below.

```js
/**
  * @param {Object} foo
  * @param {number} foo.a
  * @param {number} foo.b
  * @param {Object} bar
  * @param {number} bar.a
  * @param {number} bar.b
*/
function f(foo, bar) {
}
```

![capture](https://cloud.githubusercontent.com/assets/715219/6000740/7320b1be-aad9-11e4-8e93-4b9493e9dc96.JPG)

**Comment by hegemonic** [comment]
Well, that's no good! Thanks for the report--looking into it.

**Comment by hegemonic** [comment]
Fixed on master and releases/3.3. The fix will be included in JSDoc 3.3.0.

Again, thanks for the heads up about this!
